src/main/java/frc/robot/commands/gpm/IndexerFeed.java

```diff
@@ -1,31 +1,36 @@
 package frc.robot.commands.gpm;
 
+import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.gpm.StorageIndex;
 
-public class IndexerFeed {
+public class IndexerFeed extends Command {
 
     private final StorageIndex m_index;
 
     public IndexerFeed(StorageIndex index){
         this.m_index = index;
     }
 
+	@Override
     public void initialize (){
 
         m_index.ejectIntoShooter();
 
     }
 
+	@Override
     public void execute(){
 
     }
 
+	@Override
     public boolean isFinished(){
 
         return !m_index.hasNote();
         
     }
 
+	@Override
```

Comment:
These don't need to be indented. You don't have to change them back for this PR, but in the future you could leave them unindented.